easybuild/easyconfigs/x/xtensor/xtensor-0.25.0-foss-2023a.eb

```diff
@@ -0,0 +1,52 @@
+##
+# Authors::   Guillermo Piccoli <guillermo.piccoli@hpcnow.com>
+##
+
+easyblock = 'Bundle'
+
+name = 'xtensor'
+version = '0.25.0'
+
+homepage = 'https://github.com/xtensor-stack/xtensor'
+description = "xtensor is a C++ library meant for numerical analysis with multi-dimensional array expressions."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
I don't see any mpi parts here, so we can take the opportunity to move this to gfbf level (like with SciPy-bundle)